command/server/config_test_helpers.go

```diff
@@ -503,8 +503,8 @@ func testUnknownFieldValidation(t *testing.T) {
 			Problem: "unknown or unsupported field bad_value found in configuration",
 			Position: token.Pos{
 				Filename: "./test-fixtures/config.hcl",
-				Offset:   583,
-				Line:     34,
+				Offset:   652,
+				Line:     37,
```

Comment:
Looks like an extra change snuck in here, or is this intentional?

Reply:
It was intentional, the test code is leveraging the file position markers within `command/server/test-fixtures/config.hcl` so it needed to be updated. The same fix was done on 1.15/main